coil-svg/src/main/java/coil/decode/SvgDecoder.kt

```diff
@@ -0,0 +1,32 @@
+@file:Suppress("DEPRECATION", "unused")
+
+package coil.decode
+
+import android.graphics.drawable.PictureDrawable
+import coil.bitmappool.BitmapPool
+import coil.size.PixelSize
+import coil.size.Size
+import com.caverock.androidsvg.RenderOptions
+import com.caverock.androidsvg.SVG
+import com.caverock.androidsvg.SVGParseException
+import okio.BufferedSource
+
+/**
+ * A [Decoder] that uses android-SVG to decode SVG files.
```

Comment:
Looks like they style it as `AndroidSVG` + I would link to their site: `A [Decoder] that uses [AndroidSVG](http://bigbadaboom.github.io/androidsvg/) to decode SVG files.`